cli/admin/src/test/java/io/pravega/cli/admin/controller/TLSEnabledControllerCommandsTest.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.admin.controller;
+
+import io.pravega.cli.admin.AbstractTlsAdminCommandTest;
+import io.pravega.cli.admin.utils.TestUtils;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.admin.ReaderGroupManager;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.ReaderGroupConfig;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.StreamConfiguration;
+import lombok.Cleanup;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.UUID;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class TLSEnabledControllerCommandsTest extends AbstractTlsAdminCommandTest {
+
+    @Before
+    @Override
+    public void setUp() throws Exception {
+        tlsEnabled = true;
+        super.setUp();
+    }
+
+    @After
+    @Override
+    public void tearDown() throws Exception {
+        super.tearDown();
+    }
+
+    @Test
+    public void testAllCommands() throws Exception {
```

Comment:
This test is too ambitious. It tests multiple behaviors in a single test. If one behavior fails later if something is changed, it'll mask failures in other behaviors. Refactor this into multiple tests.

Reply:
Resolved all ambitious tests